tests/test_elm_client.py

```diff
@@ -44,17 +44,34 @@ def test_that_it_works(self):
 
 
 class TestElmPackage(unittest.TestCase):
-    def test_that_it_works(self):
+    def test_with_query_params(self):
```

Comment:
If you decouple the `elm_package_json` into `needs_query_strings` then unit testing also becomes smaller for each respective function.